src/common/PickKeysByType.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * Pick only the keys that match the Type `U`
+ */
+export type PickKeysByType<T, U> = string &
```

Comment:
what is the point of `string &` here? it completely makes useless the next code.

Reply:
I found out the issue in case when field in the model has custom column name in the database. For example, field name `balanceValue`, but in the table it is `balance_value`. TS force me set `balanceValue` but generated SQL contains the wrong field at the end.